lib/dialects/abstract/query-generator.js

```diff
@@ -2134,7 +2133,7 @@ class QueryGenerator {
         if (_.isPlainObject(arg)) {
           return this.whereItemsQuery(arg);
         }
-        return this.escape(arg);
+        return this.escape(typeof arg === 'string' ? arg.replace('$', '$$$') : arg);
```

Comment:
Just one question, out of curiosity, why `$$$`? I would have guessed `$$`.

Reply:
The replacement string with special replacement patterns
`$$ | Inserts a "$".`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace
`> a = '$sequelize'
'$sequelize'
> a.replace
[Function: replace]
> a.replace('$', '$$')
'$sequelize'
> a.replace('$', '$$$')
'$$sequelize'`

Reply:
Ahhh, of course, `$` has a special meaning within the JS escape function, so you had to escape the escape so that the escape escapes correctly :)